asset/file_exporter/processor.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const {
+    BatchProcessor
+} = require('@terascope/job-components');
+const json2csv = require('json2csv').parse;
+const Promise = require('bluebird');
+const path = require('path');
+const fs = require('fs');
+const { TSError } = require('@terascope/utils');
+
+class FileBatcher extends BatchProcessor {
+    constructor(context, opConfig, executionConfig) {
+        super(context, opConfig, executionConfig);
+        this.client = Promise.promisifyAll(fs);
```

Comment:
It is weird to call this `client` when it is really just a `fs` with promises. We should probably use a pre-promisified fs library like `fs-extra` to avoid breaking changes in behaviors

Reply:
I added `fse` to the `file_exporter` and replaced `this.client` with it in both the `file_reader` and `file_exporter`